JavaAssignments/src/amarja/Equalarrays.java

```diff
@@ -0,0 +1,47 @@
+package amarja;
+import java.util.Scanner;
+//import java.util.Scanner;
+public class Equalarrays {
+	boolean findeqarr(int num1[],int num2[])
+	{	boolean flag=false;
+		if (num1.length != num2.length)
+			return flag;
+		else 
+			{
+				for (int index=0;index<num1.length;index++)
+				{
+					if (num1[index]== num2[index])
+							flag=true;
+					else
+							flag=false;
```

Comment:
break statement required here.

Reply:
Incorporated comments .Break statement added and seperate function created for input